firmware/src/main.cpp

```diff
@@ -1,67 +1,115 @@
+#include <pico/stdlib.h> // for uart printing
 #include <cstring>
+#include <stdio.h>
+#include <config.h>
+#include <pico/multicore.h>
 #include <harp_c_app.h>
 #include <harp_synchronizer.h>
 #include <core_registers.h>
 #include <reg_types.h>
-#include <bme688.h>
-#include <config.h>
-#ifdef DEBUG
-    #include <pico/stdlib.h> // for uart printing
-    #include <cstdio> // for printf
-#endif
+#include "core1_sensor.h"
 
 // Create device name array.
-const uint16_t who_am_i = 1234;
+const uint16_t who_am_i = 1405;
```

Comment:
Let's put this magic number in `config.h` along with the major/minor revisions. Here's [an example](https://github.com/AllenNeuralDynamics/harp.device.treadmill-driver/blob/main/firmware/inc/config.h#L4)